po/ru.ebook-speaker.po

```diff
@@ -0,0 +1,355 @@
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: src/alsa_ctl.c:38
+msgid "Cannot read /proc/asound/cards"
+msgstr "Невозможно прочесть /proc/asound/cards"
+
+#: src/common.c:33 src/common.c:294 src/common.c:510 src/ebook-speaker.c:2111
+#: src/ebook-speaker.c:2122
+msgid "Please try to play this book with daisy-player"
+msgstr "Пожалуйста, попробуйте воспроизвести эту книгу с помощью Daisy-плеера"
+
+#: src/common.c:773
+#, c-format
+msgid "Cannot read type: %d"
+msgstr "Невозможно прочесть тип: %d"
+
+#: src/common.c:838
+msgid "Go to page number:"
+msgstr "Перейти на страницу: "
+
+#: src/common.c:856 src/ebook-speaker.c:1674 src/ebook-speaker.c:2627
+#: src/ebook-speaker.c:2932
+msgid "Please wait..."
+msgstr "Пожалуйста, подождите..."
+
+#: src/common.c:1096
+msgid "Select a soundcard:"
+msgstr "Выберите звуковую карту:"
+
+#: src/daisy2.02.c:49 src/daisy2.02.c:130 src/daisy2.02.c:236
+#: src/ebook-speaker.c:2514
+#, c-format
+msgid "Cannot read %s"
+msgstr "Невозможно прочесть %s"
+
+#: src/ebook-speaker.c:304
+msgid "Select a Text-To-Speech application"
+msgstr "Выберите синтезатор речи"
+
+#: src/ebook-speaker.c:322
+msgid "Maximum number of TTS's is reached."
+msgstr "Достигнут максимальный набор допустимых синтезаторов речи"
+
+#: src/ebook-speaker.c:682 src/list_dir.c:55 src/list_dir.c:251
+msgid "'h' for help"
+msgstr "'h' для справки"
+
+#: src/ebook-speaker.c:687
+#, c-format
+msgid "%d pages"
+msgstr "%d страниц"
+
+#: src/ebook-speaker.c:692
+#, c-format
+msgid "level: %d of %d"
+msgstr "Уровень: %d из %d"
+
+#: src/ebook-speaker.c:696
+#, c-format
+msgid "total phrases: %d"
+msgstr "Всего фраз: %d"
+
+#: src/ebook-speaker.c:782 src/list_dir.c:165
+msgid "These commands are available in this version:"
+msgstr "Следующие команды доступны в этой версии:"
+
+#: src/ebook-speaker.c:786 src/list_dir.c:169
+msgid "cursor down,2   - move cursor to the next item"
+msgstr "курсор вниз,2   - переместить курсор на следующий элемент"
+
+#: src/ebook-speaker.c:788 src/list_dir.c:171
+msgid "cursor up,8     - move cursor to the previous item"
+msgstr "курсор вверх,8   - переместить курсор на предыдущий элемент"
+
+#: src/ebook-speaker.c:790
+msgid "cursor right,6  - skip to next phrase"
+msgstr "курсор вправо,6     - перейти к следующей фразе"
+
+#: src/ebook-speaker.c:792
+msgid "cursor left,4   - skip to previous phrase"
+msgstr "курсор влево,4      - перейти к предыдущей фразе"
+
+#: src/ebook-speaker.c:794 src/list_dir.c:177
+msgid "page-down,3     - view next page"
+msgstr "страница вниз,3     - просмотреть следующую страницу"
+
+#: src/ebook-speaker.c:796 src/list_dir.c:179
+msgid "page-up,9       - view previous page"
+msgstr "страница вверх,9     - просмотреть предыдущую страницу"
+
+#: src/ebook-speaker.c:798
+msgid "enter           - start playing"
+msgstr "enter         - начать воспроизведение"
+
```

Comment:
Please take care of the alignment of the items. These shows up in the help page, so we'd rather have the various items properly aligned.